src/terminal/color.lua

```diff
@@ -0,0 +1,303 @@
+-- Module for setting text colors and attiributes
+-- contains other functionality such as enabling/disabling text attributes like underline, blinking
+-- and reversing. Also contains funcitionality for setting brightness levels.
+-- @usage
+-- local color = require("terminal.color")
+
+-- @module terminal.color
+-- Text colors and attributes.
+-- Managing the text color and attributes.
+-- @section textcolor
```

Comment:
```suggestion
--- Module for setting text colors and attributes
-- contains other functionality such as enabling/disabling text attributes like underline, blinking
-- and reversing. Also contains functionality for setting brightness levels.
-- @module terminal.color
-- @usage
-- local color = require("terminal.color")
```

a triple-dash is the marker for a doc comment. So the first line of a doc-comment needs to start with that.